app/controllers/pools_controller.rb

```diff
@@ -0,0 +1,26 @@
+class PoolsController < ApplicationController
+  def create
+    name = "Pool #{Tournament.find(pool_params[:tournament_id]).pools.count + 1}"
+    @pool = Pool.create(name: name, tournament_id: pool_params[:tournament_id])
+    if @pool.save
+      redirect_to tournament_pools_path(@pool.tournament_id)
+    else
+      flash.now[:errors] = @pool.errors.full_messages
+      render :new
```

Comment:
This will fail because you don't have `new` method here. You should redirect back to the page where you call this `create` method.